tasks/custom_checks.yml

```diff
@@ -0,0 +1,11 @@
+---
+# Install custom check scripts, if any.
+- name: Template out custom service checks
```

Comment:
This only puts a custom check, doesn't remove any should the user alter `nagios_custom_checks` later on.

Reply:
That's correct.  The set of checks to actually run is defined elsewhere which is where you can turn them on and off.

Reply:
Ah so you would use `nagios_custom_checks` to install scripts, and then in some other variable, say hey now use this check? That isn't in the readme. I'm now thinking that maybe the name of the variable should change to make it more clear that its for installing scripts, not controlling checks to be run against things.